src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day--mon calendar--month-length--31">
+        <div class="calendar__day calendar__day--1"></div>
+        <div class="calendar__day calendar__day--2"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
```

do not use extra styles in this task